### PR TITLE
Refactor RNTester to show individual tests on different page.

### PR DESCRIPTION
**Pull request by dotnetjunky**
- Showing the list of available tests on the left column.
- When a test is selected, it is shown in the right panel.

### CODE REVIEW

File: experiments/react-native-uifabric/src/RNTester/FabricTester.tsx [hunk 1]
```diff
@@ -19,70 +14,66 @@ msgq.spy(true);
...
+
+// sort tests alphabetically by name 
+const sortedTestComponents = allTestComponents.sort(
+  (a, b) => a.name.localeCompare(b.name)
+);
+
+const TestListSeparator = Separator.customize(
+  {
+    tokens: {
+      color: 'darkGray',
```
Comment (tkamireh) [inline experiments/react-native-uifabric/src/RNTester/FabricTester.tsx]: for fun, you could try using one of the themed colors

File: experiments/react-native-uifabric/src/RNTester/FabricTester.tsx [hunk 2]
```diff
@@ -19,70 +14,66 @@ msgq.spy(true);
...
-  const theme = useTheme();
-  return (
-    <ReactNative.View style={[styles.viewStyle, styles.stackStyle, { backgroundColor: theme.colors.background }]}>
-      <PrimaryButton onPress={onClick} content="Primary Button" disabled={disabled} />
-      <Button onPress={onClick} content="Default Button" disabled={disabled} />
-      <StealthButton onPress={onClick} content="Stealth Button" disabled={disabled} />
-      <Text>This is a text element</Text>
-      <Button onPress={onClick} content="This button has longer text" disabled={disabled} />
-    </ReactNative.View>
+export interface IFabricTesterProps {
```
Comment (tkamireh) [inline experiments/react-native-uifabric/src/RNTester/FabricTester.tsx]: You don't actually need to export this right?

Comment (tkamireh) [reply experiments/react-native-uifabric/src/RNTester/FabricTester.tsx]: nvm, though what's the expected usage of RNTesterApp? How are we supposed to pass props in?

File: experiments/react-native-uifabric/src/RNTester/TestComponents/ThemeTest.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import * as ReactNative from 'react-native';
+import { ThemeProvider, useTheme } from '@uifabricshared/theming-react-native';
+import { commonTestStyles } from '../styles';
+import { Button, PrimaryButton, StealthButton, Text } from '../../components';
+
+const Panel: React.FunctionComponent = () => {
+    const [disabled, setDisabled] = React.useState(false);
+    const onClick = React.useCallback(() => setDisabled(!disabled), [disabled, setDisabled]);
```
Comment (tkamireh) [inline experiments/react-native-uifabric/src/RNTester/TestComponents/ThemeTest.tsx]: Is memoizing on `disabled` not sufficient?

File: experiments/react-native-uifabric/src/RNTester/FabricTester.tsx (hunk 1 above)
Comment (dotnetjunky) [reply experiments/react-native-uifabric/src/RNTester/FabricTester.tsx]: Good idea. I'll do that.

File: experiments/react-native-uifabric/src/RNTester/FabricTester.tsx (hunk 2 above)
Comment (dotnetjunky) [reply experiments/react-native-uifabric/src/RNTester/FabricTester.tsx]: I'm making changes to ReactTest in devmain to make it possible to pass in initialProps to plugins. Will send out a CR later today.

File: experiments/react-native-uifabric/src/RNTester/TestComponents/ThemeTest.tsx (hunk 3 above)
Comment (dotnetjunky) [reply experiments/react-native-uifabric/src/RNTester/TestComponents/ThemeTest.tsx]: This code is moved as-is from the existing code in `FabricTest.tsx`, line 40. I'm just moving it out to a new file.

**Pull request merged**